src/google/protobuf/compiler/csharp/csharp_enum.cc

```diff
@@ -49,8 +49,8 @@ namespace protobuf {
 namespace compiler {
 namespace csharp {
 
-EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor) :
-    SourceGeneratorBase(descriptor->file()),
+EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor, const Options* options) :
```

Comment:
I thought of the `&` option but kept it consistent.

Let's keep the cleanup for later